src/test/aws/construct/static-asset-deployment.test.ts

```diff
@@ -22,91 +19,202 @@ const testStackProps = {
   stackName: 'test',
   stage: 'test',
   stageContextPath: 'src/test/aws/common/cdkEnv',
-}
-
-class TestCommonStack extends CommonStack {
-  declare props: TestStackProps
+} as any as AppProps
 
-  constructor(parent: cdk.App, name: string, props: cdk.StackProps) {
-    super(parent, name, props)
+const defaultProps = {
+  staticAssetBucket: {
```

Comment:
These should all be from a properties json file the same way it would be when using the code. Also needs to be aligned to existing tests. If we go down this route, we need to refactor all the code and tests together for consistency and maintainability